### PR TITLE
Color updates

### PR DESCRIPTION
**Pull request by nataliafitzgerald**
Update color palette page to include pantone values and full primary palette as discussed here: https://github.com/cfpb/design-manual/issues/327#issuecomment-314831623

## Additions
- Adds pantone values for primary and secondary colors
- Adds colors to the primary palette
- CSS change to allow for 2 rows of 3 columns for primary palette
- Content tweaks 

## Removals
- No removals

## Changes
-

## Testing
-

## Review

- @cfarm 

[Preview this PR without the whitespace changes](?w=0)

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -5890,7 +5893,7 @@ figure {
     border-width: 0 15px;
     margin-right: -0.25em;
     vertical-align: top;
-    width: 50%;
+    width: 33.33333333%;
```
Comment (cfarm) [inline static/css/main.css:5896]: This was an attempt to make the 3 column layout - I will remove from the branch

**Comment by cfarm** [comment]
Added some fixes to the class names to make the secondary palette grid 4 columns. Here's a screenshot of all these changes:

screen shot 2017-08-10 at 2 14 00 pm

**Review by cfarm — APPROVED** [review]

File: brand-guidelines/color-principles.md [hunk 2]
```diff
@@ -37,14 +36,18 @@ Our core color palette includes CFPB Green, black, and eight supporting colors.
                     <h4 class="swatch_head">{{ color.name }}</h4>
                     <table class="swatch_table">
                         <tbody>
-                            <tr>
+                             <tr>
```
Comment (jimmynotjim) [inline brand-guidelines/color-principles.md:39]: Minor: Remove extra indentation here, L:47, and L:87

File: _data/cfpb-brand-colors.csv [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
-gray,dark-gray,Dark Gray,#43484e,"67, 71, 78","0, 0, 0, 87",,
-gray,gray,Gray,#5a5d61,"90, 93, 97","0, 0, 0, 77",,true
-gray,gray-80,Gray 80,#75787b,"117, 120, 123","0, 0, 0, 65",,
-gray,gray-60,Gray 60,#919395,"145, 147, 149","0, 0, 0, 50",,
-gray,gray-40,Gray 40,#b4b5b6,"180, 181, 182","0, 0, 0, 35",,
-gray,gray-20,Gray 20,#d2d3d5,"210, 211, 213","0, 0, 0, 20",,
-gray,gray-10,Gray 10,#e7e8e9,"231, 231, 233","0, 0, 0, 10",,
-gray,gray-5,Gray 5,#f7f8f9,"247, 248, 249","0, 0, 0, 5",,
-gray,white,White,#ffffff,"255, 255, 255","0,  0,  0,  0",,
+family,shortname,name,hex,RGB,PMS,CMYK,primary,secondary,,
```
Comment (jimmynotjim) [inline _data/cfpb-brand-colors.csv]: Is the empty column appended to the end of each line necessary?

**Review by jimmynotjim — CHANGES_REQUESTED** [review]

File: static/css/main.css (hunk 1 above)
Comment (jimmynotjim) [reply static/css/main.css:5896]: Why do you need to remove it? Aren't the primary colors supposed to be three columns now?

File: _data/cfpb-brand-colors.csv (hunk 3 above)
Comment (cfarm) [reply _data/cfpb-brand-colors.csv]: @nataliafitzgerald seems like you need to remove that column in Excel + resave it.

Comment (nataliafitzgerald) [reply _data/cfpb-brand-colors.csv]: @jimmynotjim @cfarm - Which column am I removing? When I view this in GH it looks like there are two empty columns to the right. Is this what we're talking about?
screen shot 2017-08-10 at 5 14 41 pm

File: brand-guidelines/color-principles.md (hunk 2 above)
Comment (nataliafitzgerald) [reply brand-guidelines/color-principles.md:39]: I believe I have now removed the indentation on L:47, and L:87.

**Review by cfarm — APPROVED** [review]
CSV looks good now!

**Pull request merged**